Composer/packages/client/src/pages/design/DebugPanel/TabExtensions/DiagnosticsTab/DiagnosticsTabHeader.tsx

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx, css } from '@emotion/core';
+import formatMessage from 'format-message';
+
+import { useDiagnosticsStatistics } from './useDiagnostics';
+import { Indicator } from './Indicator';
+
+export const DiagnosticsHeader = () => {
+  const { hasError, hasWarning } = useDiagnosticsStatistics();
+
+  const indicator = hasError ? (
+    <Indicator color={'#EB3941'} size={5} />
```

Comment:
Use fluent colors

Reply:
It's a non-standard color according to design...